### PR TITLE
feat(frontend): generalize InfoEthereum component

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

Since we are going to reuse it even for ETH/ERC20 tokens, we generalize the usage of `InfoEthereum` component.

# Changes

- Change `InfoEthereum` to accept `token` and `ckTokenSymbol` as props.

# Tests

No changes in local replica.

### CODE REVIEW

**Review by peterpeterparker — COMMENTED** [review]
- A component dedicated to ethereum should not be moved in `$lib`
- Please split the PR in two as it seems to achieve two different goals

**Comment by AntonioVentilii-DFINITY** [comment]
- moved to `icp-eth`, since it will be used both by ethereum tokens and ck-tokens, ok?
- Not really sure what you mean: this PR should be about using `InfoEthereum` not only for `icp` but for `eth` too

**Comment by peterpeterparker** [comment]
Moving the module to another folder is a refactoring.
Chaging the component to accept `token` and `ckTokenSymbol` is a feature.

**Comment by AntonioVentilii-DFINITY** [comment]
Ah! Understood! Let me use this one to change the component (I will move it in another PR)

**Comment by AntonioVentilii-DFINITY** [comment]
@peterpeterparker done, ready for another review

File: src/frontend/src/icp/components/info/InfoEthereum.svelte [hunk 1]
```diff
@@ -7,10 +7,12 @@
 	import HowToConvertEthereumModal from '$icp/components/convert/HowToConvertEthereumModal.svelte';
 	import { initSendContext, SEND_CONTEXT_KEY, type SendContext } from '$icp-eth/stores/send.store';
 	import { setContext } from 'svelte';
-	import { ckEthereumTwinToken, ckEthereumTwinTokenNetwork } from '$icp-eth/derived/cketh.derived';
 	import { replaceOisyPlaceholders, replacePlaceholders } from '$lib/utils/i18n.utils';
 	import { i18n } from '$lib/stores/i18n.store';
-	import { tokenWithFallback } from '$lib/derived/token.derived';
+	import type { Token } from '$lib/types/token';
+
+	export let token: Token;
```
[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/icp/components/info/InfoEthereum.svelte:14]: it will always be the `twinToken` when using this component, make sense! changed

**Pull request merged**